conversations.yaml

```diff
@@ -0,0 +1,536 @@
+                          mids: 6482345-345345-24525-4542, 6987364-24545-2452452-5234
+                          status: delivered
+
+      responses:
+        200:
+          description: The request was processed as expected
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+        500:
+          description: Internal server error
+      security:
+        - conversationsOauth: []
+    get:
+      tags:
+        - Conversations Services
+      description:
+        Lists all available Conversations accessible to the
+        Channel Adapter.
+      parameters:
+        - in: query
+          name: f.labels
+          description: filters by the label property
+          schema:
+            type: string
+          required: false
+        - in: path
+          name: channel-guid
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description:
+            Returns the list of available conversations.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConversationsList'
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+        500:
+          description: Internal server error
+      security:
+        - conversationsOauth: []
+components:
+  schemas:
+    ConversationsList:
+      properties:
+        items:
+          type: array
+          items:
+            properties:
+              guid:
+                type: string
+                format: uuid
+              title:
+                type: string
+              description:
+                type: string
+              keyword:
+                type: array
+                items:
+                    type: string
+              labels:
+                type: array
+                items:
+                    type: string
+              created_at:
+                type: string
+                format: date-time
+                description: "The date of creation in ISO-8601 UTC format (ex: yyyy-mm-ddThh:mm:ssZ)."
+              updated_at:
+                type: string
+                format: date-time
+                description: "The last modification date in ISO-8601 UTC format (ex: yyyy-mm-ddThh:mm:ssZ)."
+        _total:
+          type: integer
+          format: int64
+    CustomerProfileData:
+      properties:
+        first_name:
+          type: string
+        last_name:
+          type: string
+        profile_pic:
+          type: string
+          description: a publicly accessible image URL.
+        locale:
+          type: string
+          description: the ISO-639 or ISO-3166 country code.
+        timezone:
+          type: integer
+          description: "Time zone offset, the number relative to GMT, (ex: -8)."
+        phone_number:
+          type: string
+          description: must be an E.164 format number.
```

Comment:
Can you add a `pattern:` to this block to call out the E.164 format?  I don't know how complex E.164 is and whether it's possible.